androminion/res/values-de/strings-ui.xml

```diff
@@ -241,6 +243,11 @@
 	<string name="envious_initial">\u0020\u0020N\u0020\u0020</string>
 	<string name="lostInTheWoods_initial">\u0020\u0020V\u0020\u0020</string>
 	<string name="druid_boons_header">- Beiseite gelegte Segen -</string>
+	<string name="flag_initial">\u0020\u0020\u2691\u0020\u0020</string>
+	<string name="horn_initial">\u0020\u0020H\u0020\u0020</string>
+	<string name="key_initial">\u0020\u0020K\u0020\u0020</string>
```

Comment:
Is there some way to distinguish the Key (Schlüssel) and Treasure Chest (Schatzkiste) in a shortened form? Perhaps `S` for  Schlüssel and `SK` for Schatzkiste? I don't know German well enough here to know what would work.